omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ClobDatatypeStorageController.java

```diff
@@ -50,19 +50,22 @@ public String create(@RequestParam MultipartFile file, HttpServletRequest reques
 	public void retrieve(@PathVariable("uuid") String uuid, HttpServletRequest request, HttpServletResponse response)
 	        throws Exception {
 		ClobDatatypeStorage clobData = datatypeService.getClobDatatypeStorageByUuid(uuid);
-		
-		PrintWriter writer = null;
-		try {
-			writer = response.getWriter();
-			writer.print(clobData.getValue());
-			writer.flush();
-		}
-		finally {
-			if (writer != null) {
-				writer.close();
-			}
-		}
-		
+
+        if(clobData == null) {
```

Comment:
@teleivo by the way you need to fix CRLF line endings in your dev environment. It creates problems when people working with linux/osx checkout the project.